api/src/test/java/io/minio/MinioClientTest.java

```diff
@@ -551,7 +551,7 @@ public void testPutSmallObject()
     String inputString = HELLO_WORLD;
     ByteArrayInputStream data = new ByteArrayInputStream(inputString.getBytes(StandardCharsets.UTF_8));
 
-    client.putObject(BUCKET, "key", data, 11, APPLICATION_OCTET_STREAM);
+    client.putObject(BUCKET, "key", data, Long.valueOf(11), null, null, APPLICATION_OCTET_STREAM);
```

Comment:
You would need to fix `L` prefix everywhere.

Reply:
done